app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,23 +1,21 @@
-import React, { useState, useEffect } from "react";
+import React, { useEffect, useState } from "react";
 
-import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Header, SubHeader } from "neetoui/layouts";
+import { Plus, Search } from "@bigbinary/neeto-icons";
+import { Button } from "@bigbinary/neetoui/v2";
+import { Container, Header } from "@bigbinary/neetoui/v2/layouts";
+import { Input, PageLoader } from "neetoui";
 
-import notesApi from "apis/notes";
-import EmptyState from "components/Common/EmptyState";
-
-import DeleteAlert from "./DeleteAlert";
+import Card from "./Card";
+import Menubar from "./MenuBar";
 import NewNotePane from "./NewNotePane";
-import NoteTable from "./NoteTable";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
```

Comment:
Let's follow a convention of naming boolean states as follows - 
`const [isLoading, setIsLoading] = useState(true);`